test/e2e/util/mongotester/mongotester.go

```diff
@@ -120,6 +120,17 @@ func (m *Tester) HasFCV(fcv string, tries int, opts ...OptionApplier) func(t *te
 	return m.hasAdminParameter("featureCompatibilityVersion", map[string]interface{}{"version": fcv}, tries, opts...)
 }
 
+func (m *Tester) ScramIsConfigured(tries int, opts ...OptionApplier) func(t *testing.T) {
+	return m.hasAdminParameter("authenticationMechanisms", primitive.A{"SCRAM-SHA-256"}, tries, opts...)
+}
+
+func (m *Tester) EnsureAuthentication(tries int, opts ...OptionApplier) func(t *testing.T) {
```

Comment:
Should this function be called "EnsureAuthenticationIsConfigured"? because the actual process of client authentication is not being performed, but only checked if configured

Reply:
yep good point, will update to be more clear